components/service-operator/examples/sqs.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: queue.govsvc.uk/v1beta1
+kind: SQS
+metadata:
+  name: sqs-sample
+  labels:
+    group.access.govsvc.uk: gsp.examples.test
```

Comment:
I'm curious about the choice to use labels for this. It's not wrong (I don't think)...I'm just curious.